actions/get_current_timestamp_MOD.js

```diff
@@ -0,0 +1,57 @@
+module.exports = {
+  name: 'Get Current Timestamp',
+  section: 'Other Stuff',
+  meta: {
+    version: '2.1.7',
+    preciseCheck: false,
+    author: 'Itz T0kyoo',
+    authorUrl: 'https://github.com/ItzT0kyooFR',
+    downloadURL: 'https://github.com/dbm-network/mods/blob/master/actions/convert_timestamp_to_date_MOD.js',
+  },
+
+  subtitle() {
+    return `Store current timestamp`;
+  },
+
+  variableStorage(data, varType) {
+    if (parseInt(data.storage, 10) !== varType) return;
+    return [data.varName, 'Date'];
+  },
+
+  fields: ['storage', 'varName'],
+
+  html() {
+    return `
+    <div style="float: left; width: 60%; padding-top: 8px;">
+      <p><u>Note:</u><br>
+      This action store the current timestamp.</p>
```

Comment:
This action stores the current time through a timestamp. not exactly necessarily but grammar.